image_width: 1920
image_height: 1080
camera_name: narrow_stereo
camera_matrix:
  rows: 3
  cols: 3
  data: [1048.9128 ,    0.     ,  938.53824,
            0.     , 1044.4716 ,  495.39543,
            0.     ,    0.     ,    1.     ]
        # Tank Test Oct 17
        #[1096.404968, 0.0, 936.318685, 
        #0.0, 1094.906739, 502.602195, 
        #0.0, 0.0, 1.0]
        # Tank test March 29
        #[1082.021095455364, 0.0, 953.2475359884882, 
        #0.0, 1082.4040564477737, 505.5529079933963, 
        #0.0, 0.0, 1.0]
        # Field Test Jun10/11
        #[1048.9128 ,    0.     ,  938.53824,
        #    0.     , 1044.4716 ,  495.39543,
        #    0.     ,    0.     ,    1.     ]
distortion_model: plumb_bob
distortion_coefficients:
  rows: 1
  cols: 5
  data: [-0.028647, 0.001474, -0.007251, -0.003957, 0.000000] #Tank test Oct17 [0.028291565793936692, -0.006851357288034091, -0.004748946315846969, -0.004993014683900631, 0.0] #Tank test March 29 [0.058398861256271625, -0.02621513239047414, -0.0022150658143925287, -2.3523994306912708e-06, 0.0] # Field Test Jun10/11 [-0.028647, 0.001474, -0.007251, -0.003957, 0.000000]
rectification_matrix:
  rows: 3
  cols: 3
  data: [1., 0., 0.,
         0., 1., 0.,
         0., 0., 1.]
projection_matrix:
  rows: 3
  cols: 4
  data: [1025.0364 ,    0.     ,  925.73027,    0.     ,
            0.     , 1034.43093,  488.16203,    0.     ,
            0.     ,    0.     ,    1.     ,    0.     ]
